packaging/pack.rb

```diff
@@ -9,6 +9,8 @@
 ROOT = File.join(__dir__, "..")
 DIST = File.join(ROOT, "dist")
 
+PYTHON_PLATFORMS = ["linux", "darwin", "freebsd", "openbsd", "windows"].product(["x86_64", "arm64"])
```

Comment:
If pypi doesn't allow to publish openbsd and freebsd packages, let's disable those platforms then.